src/global/AssetsImages.js

```diff
@@ -7,6 +7,7 @@ const eth = require('../assets/images/ethereum_logo.png');
 const qrColor = require('../assets/images/QR_icon.png');
 const bitLogo = require('../assets/images/bitnation_logo.png');
 const bitLogoBig = require('../assets/images/bitnation_logo_big.png');
+const walletIcon = require('../assets/images/wallet.png');
```

Comment:
I guess it's not used anymore, so you can remove it back as well as the image file itself.